src/coffee/directives/api/managers/spiderfier-marker-manager.coffee

```diff
@@ -0,0 +1,86 @@
+angular.module('uiGmapgoogle-maps.directives.api.managers')
+.factory 'uiGmapSpiderfierMarkerManager', ['uiGmapLogger',
+'uiGmapFitHelper', 'uiGmapPropMap', 'uiGmapMarkerSpiderfier', ($log, FitHelper, PropMap, MarkerSpiderfier) ->
+  class SpiderfierMarkerManager
+    @type = 'SpiderfierMarkerManager'
+    constructor: (gMap, opt_markers={}, @opt_options = {}, @opt_events) ->
+      @type = SpiderfierMarkerManager.type
+
+      @clusterer = new MarkerSpiderfier gMap, @opt_options
+
+      @propMapGMarkers = new PropMap() #keep in sync with cluster.markers_
+
+      @attachEvents @opt_events, 'opt_events'
+
+      @noDrawOnSingleAddRemoves = true
+      $log.info @
+
+    checkKey: (gMarker) ->
+      unless gMarker.key?
+        msg = 'gMarker.key undefined and it is REQUIRED!!'
+        $log.error msg
+
+    add: (gMarker)=>
+      gMarker.setMap @clusterer.map #puts on map (could optimize for draw.. ugh)
```

Comment:
Big difference compared to marker clutterer is here:

The OMS does not add the marker to the map; it expects it to already be done. Therefore there is no drawing function.